pyproject.toml

```diff
@@ -2,6 +2,134 @@
 requires = ["setuptools", "wheel", "setuptools-rust"]
 build-backend = "setuptools.build_meta"
 
+[project]
+name = "qiskit"
+description = "An open-source SDK for working with quantum computers at the level of extended quantum circuits, operators, and primitives."
+requires-python = ">=3.8"
+license = { file = "LICENSE.txt" }
+authors = [
+    { name = "Qiskit Development Team", email = "hello@qiskit.org" },
+]
+keywords = [
+    "qiskit",
+    "quantum circuit",
+    "quantum computing",
+    "quantum programming language",
```

Comment:
These are good additions. While we're modifying the metadata, did you want anything compiler related here?

Reply:
Oh, I think I just made sure it was synchronised with the tags on the GitHub repo, rather than going out my way to add things.  We can add "quantum compiler" to both if you thing it'd be good?

Reply:
Nah it's fine, we can always tweak these later if needed.